library/packages/src/lib/y2packager/product_upgrade.rb

```diff
@@ -33,7 +33,9 @@ class ProductUpgrade
       # SLED11 => SLED15
       ["SUSE_SLED"]              => "SLED",
       # SLES4SAP11 => SLES4SAP15
-      ["SUSE_SLES_SAP"]          => "SLES_SAP"
+      ["SUSE_SLES_SAP"]          => "SLES_SAP",
+      # openSUSE => SLES
+      ["openSUSE"]               => "SLES"
```

Comment:
So what happen when you upgrade with leap?

Reply:
Then the SLES will not be available and this line won't match. The mapping is `[installed products]` => `available product`.

I'll improve the comment.